doc/src/guide/listeners.asciidoc

```diff
@@ -203,6 +203,23 @@ ranch:set_max_connections(tcp_echo, MaxConns).
 
 The change will occur immediately.
 
+=== Running out of file descriptors
+
+Operating systems have limits on the number of sockets
+which can be opened by applications. When this maximum is
+reached the listener can no longer accept new connections. The
+accept rate of the listener will be automatically reduced, and a
+warning message will be logged.
+
+[source]
+=ERROR REPORT==== 13-Jan-2016::12:24:38 ===
```

Comment:
No it's fine, was just wondering if you changed something and forgot to copy paste the change. If it shows like this it can stay like this with nothing added.